package.json

```diff
@@ -14,7 +14,8 @@
     "bourbon-neat": "^1.7.2",
     "flux": "^2.0.1",
     "object-assign": "^2.0.0",
-    "react": "^0.12.2"
+    "react": "^0.12.2",
+    "react-router": "^0.12.4"
```

Comment:
The latest version of react-router requires react ^0.13 so I used an older version.